meg_runtime/locking/manager.py

```diff
@@ -0,0 +1,100 @@
+"""MEG system file locking
+
+To be used to lock files, unlock files, override locks, and view locks
+Will confirm user roles and preform required git operations
+
+All file paths are relitive to the repository directory
+Working directory should be changed by the git module
+"""
+
+import os
+from meg_runtime.locking.lockFile import LockFile
+from meg_runtime.logger import Logger
+
+LOCKFILE_DIR = ".meg" + os.sep
+LOCKFILE_NAME = "locks.json"
+
+class LockingManager:
```

Comment:
Same as the other class, you probably want to derive from at least object, but also maybe a dict here too. Also you need to make this a singleton. The others are also singletons, they don't need references. You also need to catch any exceptions in this class and log them and return appropriate values for success/failure.